ui/src/components/Pin.tsx

```diff
@@ -23,12 +45,26 @@ export function Pin({
     iconUrl: state === PinState.Unfocused ? unfocused : resting,
     iconRetinaUrl: state === PinState.Unfocused ? unfocused : resting,
     iconAnchor: state === PinState.Selected ? [29, 70] : [25, 61],
-    popupAnchor: null,
+    popupAnchor: [-234, 200],
```

Comment:
Any reason we did these numbers in particular?

Reply:
So it's bit rough, but popup Anchor is basically the offset from the pin --> horizontal offset = half of popup to middle of pin and vertical offset is half of vertical popup-half of popup (vertically). Took coordinates off Figma and also eye-balled it a bit.